Firefall DISASM Name Manager/Firefall DISASM Name Manager/ListViewColumnSorter.cs

```diff
@@ -41,33 +41,40 @@ public ListViewColumnSorter()
         }
 
         /// <summary>
-        /// This method is inherited from the IComparer interface. It compares the two objects passed using a case insensitive comparison.
+        /// This method is inherited from the IComparer interface. It compares the two objects passed using a case
+        /// insensitive comparison.
         /// </summary>
         /// <param name="x">First object to be compared</param>
         /// <param name="y">Second object to be compared</param>
-        /// <returns>The result of the comparison. "0" if equal, negative if 'x' is less than 'y' and positive if 'x' is greater than 'y'</returns>
+        /// <returns>The result of the comparison. "0" if equal, negative if 'x' is less than 'y' and positive if 'x'
+        /// is greater than 'y'</returns>
         public int Compare(object x, object y)
         {
             int compareResult;
             ListViewItem listviewX, listviewY;
+            string xColumnText, yColumnText;
```

Comment:
These should be renamed to columnTextX and columnTextY to follow the naming convention already set for this specific file/class.

Reply:
Done.